docs/key_modules/models.md

```diff
@@ -0,0 +1,170 @@
+
+| Model Platform | Representative Model| Multi-modality |
+| ----- | ----- | ----- |
+| OpenAI | gpt-4o | Y |
+| OpenAI | gpt-3.5-turbo | N |
+| Anthropic | claude-3-5-sonnet-20240620 | Y |
+| Gemini | gemini-1.5-pro | Y |
+| ZhipuAI | glm-4v | Y |
+| Nividia | nvidia/nemotron-4-340b-reward | N |
+| Ollama | https://ollama.com/library | ----- |
+| HuggingFace | https://huggingface.co/models | ----- |
+| LiteLLM | https://docs.litellm.ai/docs/providers | ----- |
+
+## Model calling template
+
+Here is the example code to use a chosen model. To utilize a different model, you can simply change three parameters the define your model to be used: `model_platform`, `model_type`, `model_config_dict` .
+
+```
+from camel.models import ModelFactory
+from camel.types import ModelPlatformType, ModelType
+from camel.configs import ChatGPTConfig
+from camel.message import BaseMessage
+
+# Define the model, here in this case we use GPT3.5
+model = ModelFactory.create(
+    model_platform=ModelPlatformType.OPENAI,
+    model_type=ModelType.GPT_3_5_TURBO,
+    model_config_dict=ChatGPTConfig().__dict__,
+)
+
+# Define an assitant message
+system_msg = BaseMessage.make_assistant_message(
+        role_name="Assistant",
+        content="You are a helpful assistant.",
+)
+
+# Initialize the agent
+ChatAgent(system_msg, model=model)
+```
+
+## Open source LLMs
+In the current landscape, for those seeking highly stable content generation, OpenAI's GPT-3.5 turbo,  GPT-4o are often recommended. However, the field is rich with many other outstanding open-source models that also yield commendable results. CAMEL can support developers to delve into integrating these open-source large language models (LLMs) to achieve project outputs based on unique input ideas.
+
+While proprietary models like GPT-3.5 turbo and GPT-4 have set high standards for content generation, open-source alternatives offer viable solutions for experimentation and practical use. These models, supported by active communities and continuous improvements, provide flexibility and cost-effectiveness for developers and researchers.
+
+You can find the in-detail code tutorial of [how to connect to LLMs](https://www.notion.so/User-Service-Agent-Discord-auto-posting-to-be-confirmed-by-Backend-team-006ee7e719a5448faa0f5987d17273e9?pvs=21), we will walk you through the process of:
+
+1. **Selecting an Open Source LLM**: Understand the strengths and capabilities of various models available in the open-source community. 
+2. **Setting Up the Environment**: Learn how to set up your development environment to integrate these models. This includes installing necessary libraries, configuring runtime environments, and ensuring compatibility with your project requirements.
+3. **Model Integration**: Step-by-step guidance on integrating your chosen LLM into your development workflow. We will cover API configurations, and leveraging pre-trained weights to maximize model performance for tasks.
```

Comment:
what's the additional content for this part? the link seems also wrong

Reply:
I dont think we have started this part. Who wrote this part of code? 
I have a question, is the OLLAMA integration here a part of the "how to connect to LLMs"?